app/src/main/java/com/axiel7/anihyou/data/repository/MediaListRepository.kt

```diff
@@ -73,9 +76,12 @@ class MediaListRepository @Inject constructor(
             progress = newProgress,
             status = newStatus,
             startedAt = LocalDate.now().takeIf {
-                isPlanning || !entry.progress.isGreaterThanZero()
+                (!isRepeating || entry.startedAt?.fuzzyDate?.isNull() ?: true) &&
```

Comment:
`entry.startedAt == null` is not enough because an entry with no date will have a FuzzyDate(null,null,null) value